src/pages/index.astro

```diff
@@ -1,50 +1,73 @@
 ---
-import HomeLayout from '../layouts/HomeLayout.astro';
+import BaseLayout from '../layouts/BaseLayout.astro';
+import InterledgerIcon from '../components/logos/InterledgerIcon.astro';
+import OpenPaymentsIcon from '../components/logos/OpenPaymentsIcon.astro';
+import RafikiIcon from '../components/logos/RafikiIcon.astro';
 ---
-<HomeLayout title="Developers">
+<BaseLayout title="Developers">
   <main>
     <section class="hero">
       <div class="content-wrapper">
-        <div class="hero-text">
-          <h2>Interledger Developers Portal</h2>
-          <p>Enable seamless exchange of value across payment networks.</p>
-          <div class="hero__cta">
-            <a class="hero__btn" href="/developers/get-started">Read the documentation</a>
-            <a class="hero__outlink" href="https://github.com/interledger">Contribute on Github</a>
-          </div>
+        <div class="text-wrapper">
+          <h1 class="hero__heading heading--2">Interledger Developers Portal</h1>
+          <p class="hero__txt">Help shape the future of interoperable payments and enable seamless exchange of value across payment networks.</p>
+          <a class="hero__btn" href="/developers/get-involved">Get involved</a>
         </div>
-        <div class="hero-image">
+        <div class="image-wrapper">
           <img src="/developers/img/devs.svg" alt="Illustration of developers working at a table" class="hero__img" />
         </div>
       </div>
     </section>
 
+    <section>
+      <div class="content-wrapper">
+        <p class="heading--4">Learn more about the various open-source projects that Interledger is working on.</p>
+      </div>
+    </section>
+
     <section>
       <div class="content-wrapper cards">
         <div class="card">
-          <a href="/developers/rfcs/interledger-protocol">
-            <h3>Specifications</h3>
-            <p>Interledger specifications</p>
-          </a>
+          <h2 class="heading--6"><OpenPaymentsIcon /> Open Payments</h2>
+          <p>Open Payments is an open API standard that can be implemented by account servicing entities to facilitate interoperability in the setup and completion of payments for different use cases.</p>
+          <a href="https://github.com/interledger/open-payments">Learn more on Github</a>
```

Comment:
should these link to the Open Payments and Rafiki documentation sites instead?

Reply:
For now, the READMEs on the respective projects have better introductory content than the documentation sites. Until we come up with good content for the home pages of those sites, we will link to the Github pages for now.